db/migrate/20230921005432_create_hastags.rb

```diff
@@ -0,0 +1,10 @@
+class CreateHastags < ActiveRecord::Migration[7.0]
+  def change
+    create_table :hastags do |t|
+      t.string :name
+      t.string :tweets
```

Comment:
Wouldn't this field have to be connected to  tweets table through a `tweet_id` column